src/tabrmd-defaults.h

```diff
@@ -16,7 +16,7 @@
 #define TABRMD_ENTROPY_SRC_DEFAULT "/dev/urandom"
 #define TABRMD_SESSIONS_MAX_DEFAULT 4
 #define TABRMD_SESSIONS_MAX 64
-#define TABRMD_TCTI_CONF_DEFAULT "device:/dev/tpm0"
+#define TABRMD_TCTI_CONF_DEFAULT ((!access("/dev/tcm0", F_OK)) ? ("device:/dev/tcm0") : ("device:/dev/tpm0"))
```

Comment:
I don't like this, while it does work, its prone for a race condition bug. 

Is their ever a case where /dev/tpm0 and /dev/tcm0 even exist, couldn't this be handled with a ueventd link by the tcm service? Or perhaps could we pass an argument from TCM service to this service (env var), where it expands to the `--tcti=device:/dev/tcm0` in the service file?

Reply:
uh, hidden code in a macro like this sounds icky. and racy...

maybe extend the "device" backend driver take multiple arguments and then open the first? so that you could specify `device:/dev/tpm0,/dev/tcm0` or so as default?

Reply:
https://github.com/tpm2-software/tpm2-abrmd/pull/841
This PR seems to be implemented in this way. What is the current progress of this PR?